### ISSUE TITLE
Map: Tapping amenities doesn't clear stage card or active state of stage pin

### ISSUE THREAD

**Issue by stevecottle** [redbadger/pride-london-app#566]
## Description of issue

When I tap a stage pin on the map the stage card appears. When I then tap an amenities pin the stage card remains and the stage pin remains active. This is confusing as it's not clear my tap has registered as there is little feedback from the UI.

## Expected behaviour

I'd expect the stage card to be removed and the stage pin to become inactive when I tap an amenities pin.

I'd expect the amenities pin to have an active state.

## Steps to reproduce

Go to map
Tap stage pin
Tap amenities pin
Observe stage card isn't removed and stage pin is still active

## OS / Browser / Device model

Pixel

## Screenshots

![screenshot_20180703-142952-2](https://user-images.githubusercontent.com/11883999/42222976-9394ed9e-7ece-11e8-936a-de950cc14c04.jpg)

**Comment by abinocoe** [comment]
Do we have active state versions of the amenities markers?

**Comment by stevecottle** [comment]
@abinocoe Don't think so.

@JunBadger Did we do active states for the amenities pins?

**Comment by JunBadger** [comment]
Added active WC: zpl://screen?sid=5b3b995b82f94ee8a236e8c0&pid=5a916701f75c524181eebe68 

and active first aid: zpl://screen?sid=5b3b995c2ff711764b6bf8f0&pid=5a916701f75c524181eebe68

**Comment by stevecottle** [comment]
@abinocoe ^

**Comment by stevecottle** [comment]
Updated issue with:

OS / Browser / Model
iphoneX - just happens on Golden Square event

**Comment by abinocoe** [comment]
awesome, ty @JunBadger 
@stevecottle, when you say it just happens on Golden Square event, do you mean that this is the only stage that remains active? And to double check: which version are we testing from?

**Comment by JunBadger** [comment]
yeah, strangely @abinocoe golden sq is the only event that doesn't become inactive/ event card doesn't disappear - but only if you then click on the overlapping WC or first aid tent
